static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -140,6 +140,8 @@ define([
                     && !(config.switches.disableStickyAdBannerOnMobile && detect.getBreakpoint() === 'mobile')
                     && config.page.contentType !== 'Interactive'
                     && config.page.contentType !== 'Crossword'
+                    && !config.page.isImmersive
+                    && !config.page.isUsMinute
```

Comment:
At the moment they are but that may or may not be the case in the future and there is JS hanging off isImmersive config that shouldn't be applied to the minute template hence the separation.